eth/v1alpha1/beacon_chain.proto

```diff
@@ -246,6 +246,18 @@ message GetValidatorsRequest {
         // Optional criteria to retrieve the genesis set of validators.
         bool genesis = 2;
     }
+
+    // The maximum number of Validators to return in the response.
+    // This field is optional.
+    uint64 page_size = 3;
```

Comment:
```suggestion
    int32 page_size = 3;
```

To conform with https://cloud.google.com/apis/design/design_patterns#list_pagination